### PR TITLE
Documentation site

### PR DESCRIPTION
**Pull request by yaswanthmyneni**
Added all components to the documentation site.

Review index.html and style.html only. Leave remaining changes.

Site Link: https://ym-cl-doc-site.netlify.app/

### CODE REVIEW

**Review by AshutoshM29 — COMMENTED** [review]
Looks good to merge!
Nice Work!!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,863 @@
...
+<body>
+    <div class="wrapper">
+        <!-- Header Starts -->
+        <header class="header">
+            <div class="navbar-container">
+                <a class="navbar navbar-m-lr-1" href="#">
+                    <h1 class="navbar-m-0">LUCENT UI</h1>
+                </a>
+                <a class="navbar-m-left-auto navbar" href="#">
+                    <h5>HOME</h5>
```
Comment (Ankur9669) [inline index.html:22]: If you are using h5 before that you should also have h2 h3 h4, dont use h5 tags just for sizing of the font.
You can use h2 here and for font size you can use utilty classes

**Review by Naman-Saxena1 — CHANGES_REQUESTED** [review]
Can use object-fit: cover on your avatar img for better UI

**Review by Ankur9669 — APPROVED** [review]
The color scheme and code looks great!!!

**Pull request merged**